block/src/raw_async.rs

```diff
@@ -52,6 +52,13 @@ impl DiskFile for RawFileDisk {
     }
 }
 
+struct AioRequest {
```

Comment:
It's good to separate commits logically - but I don't think commits that just add traits or structs with nobody using them are useful.